docs/src/main/tut/reference.md

```diff
@@ -277,7 +277,7 @@ _Python_:
 
 _SQL_: `rf_is_no_data_tile`
  
-Returns true if `tile` contains only nodata. By definition returns false if cell type does not support nodata.
+Returns true if `tile` contains only nodata. By definition returns false if cell type does not support nodata. To count nodata cells or data cells, see @ref:[`no_data_cells`](reference.md#no-data-cells), @ref:[`data_cells`](reference.md#data-cells), @ref:[`agg_no_data_cells`](reference.md#agg-no-data-cells), @ref:[`agg_data_cells`](reference.md#agg-data-cells), @ref:[`agg_local_no_data_cells`](reference.md#agg-local-no-data-cells), and @ref:[`agg_local_data_cells`](reference.md#agg-local-data-cells).
```

Comment:
Keep a lookout for other instances where the distinction between `is_no_data_tile` can be explained (i.e. zero is treated specially in these boolean evaluation cases). I can see this being quite confusing.

Reply:
I added some hopefully clarifying discussion.